packages/backend/src/peripherals/database/migrations/091_update_monitor_pkey.ts

```diff
@@ -0,0 +1,28 @@
+/*
+                      ====== IMPORTANT NOTICE ======
+
+DO NOT EDIT OR RENAME THIS FILE
+
+This is a migration file. Once created the file should not be renamed or edited,
+because migrations are only run once on the production server. 
+
+If you find that something was incorrectly set up in the `up` function you
+should create a new migration file that fixes the issue.
+
+*/
+
+import { Knex } from 'knex'
+
+export async function up(knex: Knex) {
+  await knex.schema.alterTable('update_monitor', (table) => {
+    table.dropPrimary()
+    table.primary(['project_name', 'chain_id', 'unix_timestamp'])
```

Comment:
Previously we only had one entry per project in this table. Update monitor functionality relied on this fact. Won't switching to multiple entries per project break update monitor?